workflow/scripts/simulate_reads.py

```
@@ -0,0 +1,30 @@
+# define default options
+read_length = snakemake.params.get("read_length", 70)
+read_number = snakemake.params.get("read_number", 1e5)
+mutations = snakemake.params.get("mutations", 0.001)
+random_reads = snakemake.params.get("random_reads", 0.05)
+output_type = snakemake.params.get("output_type", 1)
+output_prefix = path.commonprefix(snakemake.output).rstrip(".bwa.read")
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+# get input files
+fasta = snakemake.input.get("fasta")
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)